composer.json

```diff
@@ -1,42 +1,54 @@
 {
-	"name": "scienta/doctrine-json-functions",
+    "name": "scienta/doctrine-json-functions",
     "abandoned": false,
-	"type": "library",
-	"description": "A set of extensions to Doctrine 2 that add support for json query functions.",
-	"keywords": ["doctrine", "orm", "json", "dql", "database", "mysql", "postgres", "postgresql", "mariadb", "sqlite"],
-	"license": "MIT",
-	"authors": [
-		{
-			"name": "Doctrine Json Functions Contributors",
-			"homepage": "https://github.com/ScientaNL/DoctrineJsonFunctions/contributors"
-		}
-	],
-	"require": {
-		"php": "^7.1 || ^8.0",
-		"ext-pdo": "*"
-	},
-	"require-dev": {
-		"doctrine/orm": "^2.7",
-		"doctrine/coding-standard": "^8.0 || ^9.0",
-		"phpunit/phpunit": "^8.0 || ^9.0"
-	},
-	"autoload": {
-		"psr-4": {
-			"Scienta\\DoctrineJsonFunctions\\" : "src/"
-		}
-	},
-	"autoload-dev": {
-		"psr-4": {
-			"Doctrine\\Tests\\": "tests/Doctrine/Tests",
-			"Scienta\\DoctrineJsonFunctions\\Tests\\": "tests/"
-		}
-	},
-	"extra": {
-		"branch-alias": {
-			"dev-master": "4.1-dev"
-		}
-	},
-	"suggest": {
-		"dunglas/doctrine-json-odm": "To serialize / deserialize objects as JSON documents."
-	}
+    "type": "library",
+    "description": "A set of extensions to Doctrine 2 that add support for json query functions.",
+    "keywords": [
+        "doctrine",
+        "orm",
+        "json",
+        "dql",
+        "database",
+        "mysql",
+        "postgres",
+        "postgresql",
+        "mariadb",
+        "sqlite"
+    ],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Doctrine Json Functions Contributors",
+            "homepage": "https://github.com/ScientaNL/DoctrineJsonFunctions/contributors"
+        }
+    ],
+    "require": {
+        "php": "^7.1 || ^8.0",
+        "ext-pdo": "*",
+        "doctrine/orm": "^2.10",
```

Comment:
I doubt if this is too strict. I think only a hard requirement on `~2.*` is needed. The strictness is implied by the `doctrine/dbal` requirement.